packages/fetch-roamresearch/src/index.ts

```diff
@@ -56,6 +56,9 @@ async function checkLogin(
 
   if (state === LOGIN_STATE.NEED) {
     options?.reporter?.info("Login into Roam Research...");
+    // clear input
+    await page.type('input[name="email"]','');
```

Comment:
does this really clear the output? It seems like it's is going to type `''` after the existing value (just like the lines below are typing the content after the existing value)